### DISCUSSION TITLE
Extend user model

### DISCUSSION THREAD

**Discussion by ffoerster** [statamic/cms#7106] [Q&A]
Hey everyone,

this is probably question, that just stems from my minimal laravel experience (coming from a WP, ProcessWire & Typo3 background):

I have two user providers. One is the default file provider for backend users and another that uses the eloquent provider as a separate user db for frontend authentication. Is it possible to extend the eloquent driven users with fields, that query my statamic content (also flatfile based)? E.g. a user model, where one is able to favorite entries/content .

thanks!

**Comment by edalzell** [comment]
Sure, add the fields to the database and the appropriate methods to the `User` model.
